Algorithm.CSharp/BasicTemplateFrameworkAlgorithm.cs

```diff
@@ -82,25 +82,25 @@ public override void OnOrderEvent(OrderEvent orderEvent)
         /// </summary>
         public Dictionary<string, string> ExpectedStatistics => new Dictionary<string, string>
         {
-            {"Total Trades", "3"},
+            {"Total Trades", "19"},
```

Comment:
Still doesn't feel right seeing the number of trades change due to changing the benchmark. I think this is an indication that something isn't completely right here.